src/go/wsl-helper/pkg/process/kill_windows.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright © 2023 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package process
+
+import (
+	"fmt"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+func Kill(pid int) error {
+	if pid < 1 {
+		return fmt.Errorf("cannot kill process: invalid pid: %d", pid)
+	}
+
+	// Try to re-attach to the default console
+	defer func() {
+		_, _, _ = freeConsole.Call()
+		_, _, _ = attachConsole.Call(0xFFFFFFFF)
```

Comment:
It feels like the `0xFFFFFFFF` should be a constant in the `windows` package, but I can't find anything like this :cry:

Reply:
According to the [docs](https://learn.microsoft.com/en-us/windows/console/attachconsole) it should be `ATTACH_PARENT_PROCESS (DWORD)-1` but yeah.